rpc/pb/va/va.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto2";
+
+package va;
+
+import "core/core.proto";
+
+service VA {
+	rpc IsSafeDomain(Domain) returns (Valid) {}
+	rpc PerformValidation(PerformValidationRequest) returns (ValidationRecords) {}
+}
+
+message Domain {
```

Comment:
We should follow a consistent pattern: IsSafeDomainRequest / IsSafeDomainResponse. It's verbose, but better to be consistent than concise.